build.gradle

```
@@ -178,18 +197,19 @@ void createTestTask(String name, String environments, boolean devMode, int javaV
 		}
 		group = 'execution'
 		classpath = files([
-			'build' + File.separator + 'libs' + File.separator + 'Skript-nightly.jar',
+			'build' + File.separator + 'libs' + File.separator + (junit ? 'Skript-JUnit.jar' : 'Skript-nightly.jar'),
```

Common part can be extracted
```suggestion
			'build' + File.separator + 'libs' + File.separator + (junit ? 'Skript-JUnit' : 'Skript-nightly') + '.jar',
```

---

Doesn't change the length, and I see it as more readable with the inclusion of the .jar